aclogview/CM_Communication.cs

```diff
@@ -32,6 +76,70 @@ public class CM_Communication : MessageProcessor {
         return handled;
     }
 
+    public class TalkDirectByName : Message
+    {
+        public PStringChar MessageText;
+        public PStringChar TargetName;
+
+        public static TalkDirectByName read(BinaryReader binaryReader)
+        {
+            var newObj = new TalkDirectByName();
+            newObj.MessageText = PStringChar.read(binaryReader);
+            newObj.TargetName = PStringChar.read(binaryReader);
+            return newObj;
+        }
+
+        public override void contributeToTreeView(TreeView treeView)
+        {
+            TreeNode rootNode = new TreeNode(this.GetType().Name);
+            rootNode.Expand();
+            rootNode.Nodes.Add("MessageText = " + MessageText.m_buffer);
+            rootNode.Nodes.Add("TargetName = " + TargetName.m_buffer);
+            treeView.Nodes.Add(rootNode);
+        }
+    }
+
+    public class ChannelBroadcast : Message
```

Comment:
This message/opcode is bidirectional, and has different content depending on whether it's client -> server or server -> client. This one matches the client -> server format only. Just a heads up.